toolkit/tools/pkg/imagecustomizerlib/partitionutils.go

```diff
@@ -747,6 +747,10 @@ func extractKernelCmdlineFromGrub(bootPartition diskutils.PartitionInfo, bootDir
 	grubCfgPath := filepath.Join(tmpDirBoot, bootDirPath, DefaultGrubCfgPath)
 	kernelToArgs, err := extractKernelCmdlineFromGrubFile(grubCfgPath)
 	if err != nil {
+		// Preserve os.IsNotExist check by not wrapping if file doesn't exist.
+		if os.IsNotExist(err) {
```

Comment:
Use `errors.Is(err, fs.ErrNotExist)` instead of `os.IsNotExist(err)`. Then you won't need to do this.